### DISCUSSION TITLE
Installation doesn't see RAID 1 (sees individual drives instead)

### DISCUSSION THREAD

**Discussion by imperialone** [Security-Onion-Solutions/securityonion#6103] [Unsupported Versions] [answered]
I'm attempting to install latest SO version on a RAID 1 with about 2.5TB of available storage; this is on a repurposed HP Z620 workstation. When going through the install, it sees the two HDDs that make up the RAID 1 and only lets me "select" one of those drives to install to. Doing the install on one drive breaks RAID as the RAID controller now says it's in a degraded state. I don't see anything in the SO documentation about installing to a RAID, other than that it'll do the partitioning itself.

How do I get SO to see the RAID 1 versus the individual drives during installation? I should note this is a standalone install.

I attempted to install Windows 10 to make sure it wasn't something with the RAID itself and Windows 10 sees the one RAID "drive" as expected, so there's something that SO isn't doing or something that I need to do to have SO see the same one RAID "drive" that Win 10 does. I just don't have a clue as to what since I don't see any options during installation or anything mentioned in the documentation.

**Comment by Acewiza** [comment, marked as answer]
The way I did it was to install everything on one drive, then copy over /nsm to the raid then swap mount points.

**Reply by imperialone** [reply]
Thanks for the tip! Seeing that you had the same issue I encountered, I wonder why directly installing to a RAID isn't possible.

**Reply by Acewiza** [reply]
It's not an "issue."  SO has never supported installing nsm to fake raid or mdadm as far as I know.

**Reply by imperialone** [reply]
I see, I never heard of "fake raid" before but now that all makes sense. Thank you for your feedback.
